File: tests/cold/java/Cold.java
//  https://open.kattis.com/problems/cold

import java.util.*;
import java.io.*;

public class Cold {

    Scanner in;
    PrintStream out;

    public static void main(String [] args) throws Exception {
        Cold main = new Cold();
        main.in = new Scanner(System.in);
        main.out = System.out;
        main.run();
    }

    void run() {
        read();
        solve();
        write();
    }

    int[] temperatures;

    void read() {
        int n = in.nextInt();
        temperatures = new int[n];
        for (int i = 0; i < n; ++i) {
            temperatures[i] = in.nextInt();
        }
    }

    int negatives;

    void solve() {
        negatives = 0;
        for (int i = 0; i < temperatures.length; ++i) {
            if (temperatures[i] < 0) {
                ++negatives;
            }
        }
    }

    void write() {
        out.println(negatives);
    }

}

File: tests/cold/C++/src/cold.cpp
/*
Author: Ram Basnet
Date: January 2017
Kattis - Cold-puter Science problem
https://open.kattis.com/problems/cold
*/

#include <iostream>
#include <cstring>
#include <cassert>
#include <string>
#include "util.h"

using namespace std;

void solve();

int main(int argc, char *argv[])
{

    solve();
    return 0;
}

void solve()
{
    int n, t, cold = 0;
    cin >> n;

    for (int i = 0; i < n; i++)
    {
        cin >> t;
        cold += answer(t);
    }
    cout << cold << endl;
}